desc.d.ts

```diff
@@ -0,0 +1 @@
+declare module "colorspaces"
```

Comment:
Hmm you shouldn't need this file anymore, since I've moved it in #84 and defined the typings for the module...  Can you try removing this file and seeing if everything still works?  You might need to update your `main` branch to be in-sync with the repository's `main` branch...

Reply:
Sorry about that. Don't know how desc.d.ts was left in there? Or maybe you had asked me to delete earlier it and I didn't.